src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,49 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends Manager {
+
+    private double fixedBudget;
+
+    public FixedBudgetManager(String name, double salary) {
+        super(name, salary);
+    }
+
+    public boolean isSatisfied() {
+        if (this.getSalary() > 20_000 && (this.currentBudget() < 15_000)) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean hireNewEmployee(Employee employee) {
+        if ((this.currentStuffSalary() + employee.getSalary()) <= this.getFixedBudget()) {
```

Comment:
Ja jestem zwolennikiem używania słówka this. wyłącznie w konstruktorze i getterach/setterach. Kiedyś była taka konwencja żeby lepiej było widać że zmienna do której się odwołujemy jest polem w naszej klasie jednakże teraz każde sensowne IDE pokoloruje nam zmienną i nie jest to potrzebne.
Więcej o tym [tutaj](https://softwareengineering.stackexchange.com/a/211326) 

Jeszcze nazwa metody powinna dokładnie mówić o tym co robi dana metoda, tutaj powinno być hireNewEmployeeIfCan bądź HireNewEmployeeIfPossible